python/ray/ml/storage.py

```diff
@@ -0,0 +1,183 @@
+import abc
+import subprocess
+from typing import Dict
+
+from ray import logger
+from ray.util.annotations import DeveloperAPI
+
+S3_PREFIX = "s3://"
+GS_PREFIX = "gs://"
+HDFS_PREFIX = "hdfs://"
+ALLOWED_REMOTE_PREFIXES = (S3_PREFIX, GS_PREFIX, HDFS_PREFIX)
```

Comment:
These are kept here because Tune still depends on them (the original file has been moved from `ray.util.ml_utils.cloud`)